src/main/java/core/basesyntax/service/WriteScvService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface WriteScvService {
+    void writeDataScvFile(List<FruitTransaction> fruitTransactionList, String fileName);
+
```

Comment:
remove the last empty line in all classes/interfaces/methods where it is present